cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -310,12 +310,19 @@ message TraceBounds {
 message SegYSeismicRequest {
     oneof identifier {
         Identifier seismic = 1;
-        int64 seismic_store_id = 2;
+        int64 seismic_store_id = 2;  // Need to be a data manager or tenant user, not 3rd party, to access by tracestore
```

Comment:
```suggestion
        int64 seismic_store_id = 2;  // Need to be a data manager or tenant user, not 3rd party, to access by seismic store
```
no?

Reply:
You're right, I copy pasted this comment from the new stream traces message